### PR TITLE
Add section on different ways to authenticate to the Prometheus API

### PR DESCRIPTION
**Pull request by Roadmaster**
### Summary of changes
* Explain how to create a non-bearer macaroon token with restrictions (per org, read-only)
* Explain how to pass a macaroon token in the Authorization header

This process is known internally and we share it with customers who require it but a customer pointed out it's not documented which makes it difficult to discover.

### CODE REVIEW

File: monitoring/metrics.html.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ the user icon in the lower-left of the screen.
 
 ### External or self-hosted Grafana
 
-You can also configure your Prometheus endpoint with an existing Grafana installation, or [host one on Fly.io](https://github.com/fly-apps/grafana). Either way, you set it up thusly:
+You can also configure your Prometheus endpoint with an existing Grafana installation, or [host one on Fly.io](https://github.com/fly-apps/grafana). Either way, you set it up thus:
```
Comment (andie787) [inline monitoring/metrics.html.md:115]:
```suggestion
You can also configure your Prometheus endpoint with an existing Grafana installation, or [host one on Fly.io](https://github.com/fly-apps/grafana). Either way, you can set it up like this:
```

File: monitoring/metrics.html.md [hunk 2]
```diff
@@ -388,3 +388,48 @@ processes = ["proxy"]
...
+
+```
+Authorization: Bearer THE_TOKEN
+```
+
+### Fly org-restricted or read-only token
+
+This kind of token or "[macaroon](https://fly.io/blog/macaroons-escalated-quickly/)" can be scoped to a single organization or configured to only allow read operations, which can be safer than using a full-blown read-write token that grants access to all organizations under your account.
+
+### Generating
```
Comment (andie787) [inline monitoring/metrics.html.md:408]:
```suggestion
### Generating tokens
```

File: monitoring/metrics.html.md [hunk 3]
```diff
@@ -388,3 +388,48 @@ processes = ["proxy"]
...
+An org-restricted token can be generated like so:
```
Comment (andie787) [inline monitoring/metrics.html.md:409]:
```suggestion
Create an org-restricted token:
```

File: monitoring/metrics.html.md [hunk 4]
```diff
@@ -388,3 +388,48 @@ processes = ["proxy"]
...
+
+```
+fly token create org -o THE_ORGANIZATION
+```
+
+A read-only token is created like this:
```
Comment (andie787) [inline monitoring/metrics.html.md:415]:
```suggestion
Create a read-only org-restricted token:
```

File: monitoring/metrics.html.md [hunk 5]
```diff
@@ -388,3 +388,48 @@ processes = ["proxy"]
...
+
+Note that the read-only token still has access to all organizations in your account (although it cannot modify anything, only read data). These two can be combined to create a read-only token that is restricted to a single organization:
+
+```
+fly token create readonly --from-existing -t "$(fly token create org -o THE_ORGANIZATION)"
+```
+
+These tokens look like this once generated: `FlyV1 fm2_lJPECAAAAAAAAC7txBAzYI6PRWhHLT...(a lot of base64-encoded text)`.
+
+#### Using
```
Comment (andie787) [inline monitoring/metrics.html.md:429]:
```suggestion
#### Using tokens
```

File: monitoring/metrics.html.md [hunk 6]
```diff
@@ -388,3 +388,48 @@ processes = ["proxy"]
...
+fly token create org -o THE_ORGANIZATION
+```
+
+A read-only token is created like this:
+
+```
+fly token create readonly
+```
+
+Note that the read-only token still has access to all organizations in your account (although it cannot modify anything, only read data). These two can be combined to create a read-only token that is restricted to a single organization:
```
Comment (andie787) [inline monitoring/metrics.html.md:421]: The read-only token you create with `fly token create readonly` is already an org scoped token, so you don't need to do this.

**Review by andie787 — COMMENTED** [review]
Thanks for adding this info! Left some feedback to consider.

File: monitoring/metrics.html.md (hunk 6 above)
Comment (Roadmaster) [reply monitoring/metrics.html.md:421]: Thanks, I'll remove this part entirely :)

**Pull request merged**